src/workloads/basic/ReadOnlyMultiThreaded.yml

```diff
@@ -0,0 +1,91 @@
+SchemaVersion: 2018-07-01
+Owner: Product Performance
+Description: |
+  Runs a multi-threaded, read-only workload comprising of `findOne` commands that all target the 
+  same document. The workload inserts a single document and then spawns multiple worker threads to 
+  retrieve it.
+
+GlobalDefaults:
+  Nop: &Nop {Nop: true}
+  Database: &Database test
+  Collection: &Collection test
+  PaddingLength: &PaddingLength 64 # bytes
+  WarmUpPeriod: &WarmUpPeriod 30 seconds
+  RunPeriod: &RunPeriod 5 minutes
+  Threads: &Threads 64
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 500
```

Comment:
Do you actually need a custom client here? The number of threads is less than the default [maxPoolSize](https://www.mongodb.com/docs/languages/cpp/cpp-driver/current/connection-pools/#:~:text=to%20the%20minPoolSize.-,maxPoolSize,-The%20maximum%20number) of 100.

Reply:
I have this set to 200 and 100, and Genny failed on some runs due to connection limit issues, so opted to set it to a value that I am sure is never exhausted by the benchmark. LMK if that's acceptable or if you want us to root cause the issue as part of this PR.

Reply:
That is fine ... but peculiar. Can you raise a PERF ticket to note the issues you found. I'm not sure how much effort it will get as genny is going to be mothballed but I'd like to track the issues you found.

I'm confused as to why a test with 64 threads required many more connections .. unless this test caused a mini-connection storm which would be interesting in itself.

I was going to suggest that [minPoolSize](https://www.mongodb.com/docs/languages/cpp/cpp-driver/current/connection-pools/#:~:text=to%20the%20pool.-,minPoolSize,-Sets%20a%20target) might avoid the need for a warmup but on reading the docs that doesn't seem to be the case.

Reply:
Filed [PERF-5681](https://jira.mongodb.org/browse/PERF-5681).